config/pipelines/detect_burnin.pipe

```diff
@@ -75,12 +75,23 @@ process detect_burnin__dilater
   filter:vxl_morphology:kernel_radius   = 2.0
 
 # =============================================================================
-process detect_burnin__out
+process detect_burnin__convert
   :: image_filter
   filter:type                           = vxl_convert_image
   filter:vxl_convert_image:type         = byte
   filter:vxl_convert_image:scale_factor = 255
 
+# =============================================================================
+process detect_burnin__merge
```

Comment:
I'm wondering if it makes sense to have a separate `inpaint_burnin.pipe` that includes `detect_burnin.pipe` and adds the inpainting parts.  Some users may want to use just the detector.  Another option is to have `inpaint_burnin.pipe` include just these two process and not include `detect_burnin.pipe` and then require the user to include both and connect them up.  I'm not sure which is better.

Reply:
Actually, that's probably a better idea... or even, assumes that whoever includes it has a `detect_burnin__out`. (Also, while I didn't think of it earlier, I'm not sure inpainting belongs in "**detect**_burnin.pipe".)

Reply:
I followed Matt's initial suggestion of making a `inpaint_burnin.pipe` which is a very light wrapper around `detect_burnin.pipe`. I made a small breaking change to the existing functionality which was updating `detect_burnin__out` to `detect_burnin__mask`.